app/src/main/java/com/akshay/newsapp/koin/modules/NewsDatabaseModule.kt

```diff
@@ -0,0 +1,23 @@
+package com.akshay.newsapp.koin.modules
+
+import com.akshay.newsapp.db.NewsDatabase
+import org.koin.android.ext.koin.androidApplication
+import org.koin.dsl.module
+
+/**
+ * All database related initializations
+ */
+
+val newsDatabaseModule = module {
+
+	// NewsDatabase Instance
```

Comment:
You don't need to add a comment for each dependency; this applies to all the module files.